### DISCUSSION TITLE
Follow System Suggestion

### DISCUSSION THREAD

**Discussion by Sorro** [NamelessMC/Nameless#2146] [Ideas]
So you know when you click follow on peoples page it shows who has followed who. So i was thinking of a idea what should display if people have followed someone. I have 2 ideas. 

I will call this below this text ''Idea 1''

When people view Forums in the right side it will say something like this: (This idea is like the latest posts but the people you follow)
![follow1](https://cloud.githubusercontent.com/assets/8834158/17628095/0d904cfa-60b4-11e6-9b3f-53f58a1c5a77.png)

When people view the home page it should say something like this:
![follow2](https://cloud.githubusercontent.com/assets/8834158/17628442/a254592a-60b5-11e6-95d9-75b3e5b9747d.png)

I will call this ''Idea 2''

If you dont like my first idea i have another one, its like the Enjin.com's friend activity in the bottom right :)

EDIT: I changed it with some photos so you guys can understand better

Sorry if i confused you. i will explain better if u dont understand me :)

**Comment by godleydemon** [comment]
Honestly, I'm assuming your not a native English speaker? If not, and you are a native English speaker, I couldn't make heads from tails on what in the world you are talking about lol.

**Comment by Sorro** [comment]
@godleydemon I have updated the post. I hope you understand me now :) I photoshoped how i kinda want it to look like :)

**Comment by lcswolf** [comment]
Looks cool

**Comment by Sorro** [comment]
@lcswolf ty!

**Comment by godleydemon** [comment]
ok that's ALOT more clear now, thanks for that. As for the suggestion, it's not a BAD idea persay. But I think it'd have to be a feature for 2.0, depending on what Sam thinks. The dream would be to have a draggable interface where you can put these where ever you want on an interface in the admin view, then save it and have that all appear where you wanted it on the homepage. But that's probably a LONG way off yet. I THINK at the moment, things your friends do that your following, appear in the notification box. Which is probably where they should remain for now. You don't want to make the homepage to clunky and over filled with information. 

That and the reason for the notifications like that on Enjin, is because you can have friends and yourself in more than one group. You can have them spread out all over the place. So having a notification system like that is actually handy. Not something that would be to handy for this type of architecture.

**Comment by Sorro** [comment]
Ok, but what friends do in notification box isnt there.. it shows who has posted in ur wall, applications subbmited.

And ye the dream is to have a draggable interface.
